src/Std/Tactic/BVDecide/Syntax.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Henrik Böving
+-/
+prelude
+import Init.Notation
+import Init.Simproc
+
+set_option linter.missingDocs true -- keep it documented
+
+namespace Lean.Parser
+
+namespace Tactic
+
+/--
+Close a goal by:
+1. Turning it into a BitVec problem.
+2. Using bitblasting to turn that into a SAT problem.
+3. Parsing a previously produced LRAT proof for that SAT problem.
+4. Verifying the LRAT proof using proof by reflection.
+
+It is called with the name of an LRAT file in the same directory as the current Lean file:
+```
+bv_check "proof.lrat"
+```
+-/
+syntax (name := bvCheck) "bv_check " str : tactic
+
+/--
+Close a goal by:
+1. Turning it into a BitVec problem.
+2. Using bitblasting to turn that into a SAT problem.
+3. Running an external SAT solver on it and obtaining an LRAT proof from it.
+4. Verifying the LRAT proof using proof by reflection.
```

Comment:
I'm not sure whether this docstring is too helpful for users of the tactic. In terms of implementation details, I think points 2/3/4 can be merged into something like "obtains a proof from an external SAT solver and verifies it inside Lean".

I think that the docstring should contain the following:
* Which problems the tactic can turn into BitVec problems, ideally with examples
* The information that this tactic introduces the `Lean.ofReduceBool` axiom and what this means
* That the tactic can be quite expensive to run and what to do in this case -> `bv_decide?`/`bv_check`